docs/logs.md

```diff
@@ -2,23 +2,4 @@
 
 Tailing the build logs is possible with the kpack log utility. 
 
-### Install
-
-Downloading the log utility for you operating system from the most recent [github release](https://github.com/pivotal/kpack/releases).
-
-### Usage
-
-To tail logs from all builds for an image  
-```bash
-logs -image <image-name> 
-```
-
-To tail logs from a specific build on an image  
-```bash
-logs -image <image-name> -build <build-number>
-```
-
-To tail logs from an image in a different namespace  
-```bash
-logs -image <image-name> -namespace <namespace>
-```
\ No newline at end of file
+Logs can be read with the `kp build logs` command [here](https://github.com/vmware-tanzu/kpack-cli/blob/main/docs/kp_build_logs.md).
```

Comment:
Can we silently remove logs from the next release?

Reply:
I think it would be a good idea!